config-editor/config-editor-ui/src/app/services/store/config-store.service.ts

```diff
@@ -225,12 +225,18 @@ export class ConfigStoreService {
   }
 
   reloadStoreAndRelease(): Observable<any> {
-    const testCaseMapFun = this.metaDataMap.testing.testCaseEnabled ? this.configLoaderService.getTestCases() : of({});
```

Comment:
the test specification should not change, thinking, maybe all of it or just whether test cases are enabled could be saved in the store as a variable, because it does not make sense here either to re query test specification

this can also be fed to the loader from the store when deleting a config -> no need to save anything in loader (which is better)